web/themes/custom/home_designer_theme/scss/paragraphs/about.scss

```
@@ -0,0 +1,148 @@
+.wrapper {
+  position: relative;
+  padding: 110px 0;
+  background-color: #302822;
+  z-index: 2;
+
+  border-bottom: solid 5px #b79259;
+;
+
+  @media (max-width: 1024px) {
+    padding: 50px 0;
+  }
+
+  &::before {
+    content: "";
+    background-image: url('../../images/bg/about_lamp.png');
+    background-position: left top;
+    background-repeat: no-repeat;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 0;
+    opacity: 0.7;
+  }
+
+  .container {
+    @media (max-width: 1024px) {
+      //margin-left: 10px;
+      //margin-right: 10px;
+      max-width: 1024px;
+    }
+
+    .about__block {
+      width: 100%;
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: center;
+      align-items: center;
+      position: relative;
+      z-index: 4;
+
+      @media (max-width: 800px) {
+        flex-direction: column;
+
+      }
+
+      .about__text_block {
```

```suggestion
      .about {
      &__text_block { ...
      }
      
```
could you please try to add & ?